roles/piwigo/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+piwigo_enabled: false
+piwigo_available_externally: "false"
+
+# directories
+piwigo_data_directory: "{{ docker_home }}/piwigo"
```

Comment:
include the following line in # directories:
piwigo_photos_directory: "{{ photos_root }}"